### ISSUE TITLE
Compiled Windows version is missing from releases

### ISSUE THREAD

**Issue by sanderdatema** [vranki/ExtPlane#74]
The readme mentions Linux and Windows versions are in the releases. But it's only Linux. Since it's possible to compile for those two systems using Docker, you could have Github Actions do it for you. I know you're looking for developers, but I'm afraid I can't help out with this, though I'll try to look into it.

**Comment by vranki** [comment]
Yeah, the windows cross-complilation seems to be broken at the moment for some reason. Needs to be studied. Help is welcome as I don't run X-Plane or develop on Windows.

**Comment by tannerblair** [comment]
Any updates on this? I would love to use this, but I'm less fond of signing up for an account just so I can compile it.
